app/routes/admin.settings.tsx

```diff
@@ -59,6 +68,8 @@ export const loader = async ({ request }: LoaderFunctionArgs) => {
 
       const account = await new UserModel().find(payload.id)
 
+      const categories =  await new ProductCategoryModel().findMany(0, 10)
```

Comment:
the page size and page number shouldn't be hardcoded, it should be read from the http querystring, for example:

```typescript
      const page = new URL(request.url).searchParams.get('page') || '1'
      const size = new URL(request.url).searchParams.get('size') || '20'
      const users = await new UserModel().findMany(
        Number(page) < 1 ? 0 : Number(page) - 1,
        Number(size) < 1 ? 1 : Number(size),
      )
```